packages/client/src/components/app/blocks/form/InnerFormBlock.svelte

```diff
@@ -211,17 +211,19 @@
           {/if}
         </BlockComponent>
       {/if}
-      <BlockComponent type="fieldgroup" props={{ labelPosition }} order={1}>
-        {#each fields as field, idx}
-          {#if getComponentForField(field) && field.active}
-            <BlockComponent
-              type={getComponentForField(field)}
-              props={getPropsForField(field)}
-              order={idx}
-            />
-          {/if}
-        {/each}
-      </BlockComponent>
+      {#key fields}
```

Comment:
I have a feeling I mentioned this before, but what's the purpose of the full remount of all fields whenever any field changes? I think I recommended just keying the `each` statement based on whatever property of the fields requires a full remount whenever they change.

Reply:
I ran into an issue with the `each` statement where the ordering of the list would update correctly, but some of the form components would still not reflect updates made in the settings popover. They did sometimes update, but it was inconsistent. I had a look at the `DateTimeField` and the `redrawOptions` values would only update sporadically. 

It might be possible to ensure that only the updated fields are re-rendered but keying the form fields with the `fields` object seemed to cover the field ordering and settings updates in one go.